### PR TITLE
NOISSUE  - v0.14.0 blog post

### PR DESCRIPTION
**Pull request by rodneyosodo**
### What does this do?
Submit blog post for review

### Which issue(s) does this PR fix/relate to?
No issue

### List any changes that modify/break current functionality
Magistrala v0.14.0 release blog post

### Have you included tests for your changes?
No

### Did you document any new/modified functionality?
Yes

### Notes
Not to be merged but to be reviewed for publishing to https://medium.com/abstract-machines-blog

### CODE REVIEW

**Review by arvindh123 — COMMENTED** [review]
LGTM

File: docs/release.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+### Switch to Google Zanzibar Access control approach with SpiceDB
+
+We migrated our access control system from Ory Keto to [SpiceDB](https://github.com/authzed/spicedb). SpiceDB is an open-source, [Google Zanzibar](https://authzed.com/blog/what-is-zanzibar/)-inspired database system for real-time, security-critical application permissions. Initially, we attempted to build our access control system. However, we quickly recognized the inherent complexity of this task and the importance of domain expertise. We are confident that SpiceDB is the superior solution for our requirements. The PR for this change can be found [here](https://github.com/absmach/magistrala/pull/1919).
+
+### Implement Domains for access control
+
+Magistrala now supports multi-tenancy with the introduction of domains! This exciting new feature allows for the separation of data and resources between different users or organizations. While we're still working on further refinements in future releases, we believe this marks a significant step forward for magistrala. You can find the commit for this change on [GitHub](https://github.com/absmach/magistrala/commit/c294e84d2446f8b99d22e24211da171462ecdd41).
```
Comment (Musilah) [inline docs/release.md]: Could you capitalise the first letter in magistrala.

File: docs/release.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/psTSFMoa4f4?si=Gx-Fi0hpe3RBv3jK" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+### Return Websocket support
+
+Remember web sockets? We're thrilled to announce its return to Magistrala, built from scratch for improved performance and exciting possibilities. This isn't just a revival – we're actively planning future features and even UI integration, allowing your interface to receive real-time messages directly. This opens doors for dynamic and efficient real-time communication, and we can't wait to see what you build! The PR for this change can be found [here](https://github.com/absmach/magistrala/pull/1625).
```
Comment (Musilah) [inline docs/release.md]: This is the third 'we're thrilled to announce'. Perhaps a different clause.

File: docs/release.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+### Add Invitation service
+
+Invitation service is a new feature in magistrala. This service is responsible for sending invitations to users to join a domain. This is a very important feature for multi-tenancy. We currently don't support email invitations but we are planning to add it in future releases. This is because the invitation services need to talk to the users service to get the email of the user and we don't have the communication between services yet. We are planning to add this in future releases. The commit for this change can be found [here](https://github.com/absmach/magistrala/commit/a07aabe78328d99c176f6bf6034339ce5256f13f)
```
Comment (Musilah) [inline docs/release.md]: Capitalize M in Magistrala

File: docs/release.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+### Added support for multiple event stores
+
+Magisterala takes a monumental leap forward in event handling with the introduction of multiple event store support. No longer limited to Redis, you now have the freedom to choose between NATS Jetstream and RabbitMQ alongside the familiar favourite. This decision stems from our unwavering commitment to empowering developers with powerful and user-friendly options. Using NATS or RabbitMQ as the event store you can simplify your infrastructure since it can be used as both the event store and the message broker. The PR for this change can be found [here](https://github.com/absmach/magistrala/pull/1907).
```
Comment (Musilah) [inline docs/release.md]: Wrong Magistrala spelling here

File: docs/release.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+
+### Added UI as a separate service
+
+With great enthusiasm, we present a significant upgrade to the Magistrala user interface: it's now a standalone service written in Golang! This exciting shift from a single-page Angular application brings several advantages. Firstly, maintaining a separate Golang service aligns better with our existing codebase, simplifying the development and upkeep of the UI. Secondly, this opens doors for easier extension and future functionalities. While there's still room for further development, especially regarding dashboards, this marks a major stepping stone for the UI's potential. We're actively exploring various dashboarding solutions and can't wait to share what's next! We dedicated an entire repository for the UI and it can be found [here](https://github.com/absmach/magistrala-ui).
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/psTSFMoa4f4?si=Gx-Fi0hpe3RBv3jK" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```
Comment (Musilah) [inline docs/release.md]: This image doesn't show up locally

Comment (rodneyosodo) [reply docs/release.md]: I can see it from my side in incognito mode. Are you able to see this video?

**Review by dborovcanin — DISMISSED** [review]

File: docs/blogs/index.md [hunk 6]
```diff
@@ -3,3 +3,4 @@
 This section contains a list of blogs that are related to Magistrala. They have been published to the [Official Abstract Machines blog in Medium](https://medium.com/abstract-machines-blog).
 
 - [Extending Magistrala Users Repository](./kratos.md)
+- [What's New in Magistrala v0.14.0](./v0-14-0-release.md)
```
Comment (dborovcanin) [inline docs/blogs/index.md]: Let's reorder.

File: docs/blogs/v0-14-0-release.md [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+
+We're thrilled to announce the release of Magistrala v0.14.0, packed with exciting new features and improvements that elevate your experience! This update marks a significant leap forward, making Magistrala even more powerful and versatile. Here are some of the highlights:
+
+- Switch to Google Zanzibar Access control approach with SpiceDB
+- Implement Domains for access control
+- Added UI as a separate service
+- Return Websocket support
+- Add Invitation service
+- Added support for multiple MQTT message brokers
+- Added support for multiple event stores
```
Comment (dborovcanin) [inline docs/blogs/v0-14-0-release.md]: multiple -> numerous

File: docs/blogs/v0-14-0-release.md [hunk 8]
```diff
@@ -0,0 +1,108 @@
...
+- Enable Jetstream for NATS as the default event store and message broker
+- Added support for gRPC mTLS
+- Add graceful-stop for HTTP and gRPC servers
```
Comment (dborovcanin) [inline docs/blogs/v0-14-0-release.md]: Add -> Added

File: docs/blogs/v0-14-0-release.md [hunk 9]
```diff
@@ -0,0 +1,108 @@
...
+- Logical and hard entity deletion
+- Improve tracing
+- Improve event sourcing
+- Added telemetry
+- Add prometheus and grafana for metrics
```
Comment (dborovcanin) [inline docs/blogs/v0-14-0-release.md]: added Prometheus and Grafana for metrics

File: docs/blogs/v0-14-0-release.md [hunk 10]
```diff
@@ -0,0 +1,108 @@
...
+- Return Websocket support
+- Add Invitation service
+- Added support for multiple MQTT message brokers
+- Added support for multiple event stores
+- Enable Jetstream for NATS as the default event store and message broker
+- Added support for gRPC mTLS
+- Add graceful-stop for HTTP and gRPC servers
+- Logical and hard entity deletion
+- Improve tracing
+- Improve event sourcing
```
Comment (dborovcanin) [inline docs/blogs/v0-14-0-release.md]: Improved, upgraded, use past tense.

File: docs/blogs/v0-14-0-release.md [hunk 11]
```diff
@@ -0,0 +1,108 @@
...
+
+### Return Websocket support
+
+Remember web sockets? We are ecstatic to reveal its return to Magistrala, built from scratch for improved performance and exciting possibilities. This isn't just a revival – we're actively planning future features and even UI integration, allowing your interface to receive real-time messages directly. This opens doors for dynamic and efficient real-time communication, and we can't wait to see what you build! The PR for this change can be found [here](https://github.com/absmach/magistrala/pull/1625).
+
+### Add Invitation service
+
+Invitation service is a new feature in Magistrala. This service is responsible for sending invitations to users to join a domain. This is a very important feature for multi-tenancy. We currently don't support email invitations but we are planning to add it in future releases. This is because the invitation services need to talk to the users service to get the email of the user and we don't have the communication between services yet. We are planning to add this in future releases. The commit for this change can be found [here](https://github.com/absmach/magistrala/commit/a07aabe78328d99c176f6bf6034339ce5256f13f)
```
Comment (dborovcanin) [inline docs/blogs/v0-14-0-release.md]: This is an essential feature for multi-tenancy.

File: docs/blogs/v0-14-0-release.md [hunk 12]
```diff
@@ -0,0 +1,108 @@
...
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/psTSFMoa4f4?si=Gx-Fi0hpe3RBv3jK" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+### Return Websocket support
+
+Remember web sockets? We are ecstatic to reveal its return to Magistrala, built from scratch for improved performance and exciting possibilities. This isn't just a revival – we're actively planning future features and even UI integration, allowing your interface to receive real-time messages directly. This opens doors for dynamic and efficient real-time communication, and we can't wait to see what you build! The PR for this change can be found [here](https://github.com/absmach/magistrala/pull/1625).
```
Comment (dborovcanin) [inline docs/blogs/v0-14-0-release.md]: Remember the WebSocket adapter? We are happy to reveal its return...

File: docs/blogs/v0-14-0-release.md (hunk 11 above)
Comment (dborovcanin) [inline docs/blogs/v0-14-0-release.md]: Capitalize `Invitations` service, `Users` service.

File: docs/blogs/v0-14-0-release.md [hunk 13]
```diff
@@ -0,0 +1,108 @@
...
+
+In TLS, the client verifies the server's certificate to ensure that it is connecting to the correct server. The server doesn't verify the client's certificate. Anyone who has the server's certificate can connect to the server. Mutual TLS or mTLS is a form of transport security that requires both the client and the server to present certificates to verify their identity. This is a more secure form of transport security than traditional TLS, which only requires the server to present a certificate. mTLS is a good choice for securing communication between services in a microservices architecture, where you want to ensure that both the client and the server are who they say they are. The PR for this change can be found [here](https://github.com/absmach/magistrala/pull/1848). This is enabled out-of-the-box and you don't need to do anything to enable it.
+
+### Add graceful-stop for HTTP and GRPC servers
```
Comment (dborovcanin) [inline docs/blogs/v0-14-0-release.md]: Same, use the past tense; Added.

File: docs/blogs/v0-14-0-release.md [hunk 14]
```diff
@@ -0,0 +1,108 @@
...
+
+### Add Invitation service
+
+Invitation service is a new feature in Magistrala. This service is responsible for sending invitations to users to join a domain. This is a very important feature for multi-tenancy. We currently don't support email invitations but we are planning to add it in future releases. This is because the invitation services need to talk to the users service to get the email of the user and we don't have the communication between services yet. We are planning to add this in future releases. The commit for this change can be found [here](https://github.com/absmach/magistrala/commit/a07aabe78328d99c176f6bf6034339ce5256f13f)
+
+### Added support for multiple MQTT message brokers
```
Comment (dborovcanin) [inline docs/blogs/v0-14-0-release.md]: Maybe something like: Expanded compatibility across various MQTT brokers?

**Review by dborovcanin — CHANGES_REQUESTED** [review]

**Review by dborovcanin — APPROVED** [review]

**Pull request merged**